[DOCS-1343] Add SCIM example for creating a user on multi-tenant (#1252)
wandb/docs: content/guides/hosting/iam/scim.md

### Create user

> The schema for creating a user is different for **Dedicated Cloud** and **Self-Hosted** than that for **Multi-tenant Cloud**.

Reviewer: This sentence might be confusing for some. Consider separating it into two.

Author: DOne

> The schema for creating a user is different for **Dedicated Cloud** and **Self-Hosted** than that for **Multi-tenant Cloud**.

Author: ```suggestion
Select your deployment type to view an example of creating a user using SCIM. The request and response schemas differ between **Dedicated Cloud or Self-Managed** and **Multi-tenant Cloud**.
```

| userName | String | Yes |
- **Request Example**:

    ```bash
    POST /scim/Users
    ```
    ```json
    {
    "schemas": [
        "urn:ietf:params:scim:schemas:core:2.0:User"
    ],
    "emails": [
        {
        "primary": true,

>         "value": "admin-user2@test.com"

Reviewer 2: nit: how about "dev-user2@test.com" to better match the userName and response?

Author: Done!

>         "value": "admin-user2@test.com"

Author: ```suggestion
        "value": "dev-user2@test.com"
```

            "Primary": true
        },
        "id": "def",
        "meta": {
            "resourceType": "User",
            "created": "2023-10-01T00:00:00Z",
            "location": "Users/def"
        },
        "schemas": [
            "urn:ietf:params:scim:schemas:core:2.0:User"
        ],
        "userName": "dev-user2"
    }
    ```
{{% /tab %}}

{{% tab header="Multi-tenant Cloud" %}}
In **Multi-tenant Cloud**, a user can belong to more than one organization. To "create" a user in your organization, you add them to one or more teams.
- **Endpoint**: **`<host-url>/scim/Users`**
- **Method**: POST
- **Description**: Create a new user resource.
- **Supported Fields**:

| Field | Type | Required |
| --- | --- | --- |
| emails | Multi-Valued Array | Yes (Make sure `primary` email is set) |
| userName | String | Yes |
| teams | Multi-Valued Array | Yes (the user must belong to at minimum one team) |

- **Request Example**:

    ```bash
    POST /scim/Users
    ```

    ```json
    {
    "schemas": [
        "urn:ietf:params:scim:schemas:core:2.0:User",
        "urn:ietf:params:scim:schemas:extension:teams:2.0:User"
    ],
    "emails": [
        {
        "primary": true,

>         "value": "admin-user2@test.com"

Reviewer 2: nit: also "dev-user2@test.com"

>         "value": "admin-user2@test.com"

Author: ```suggestion
        "value": "dev-user2@test.com"
```
